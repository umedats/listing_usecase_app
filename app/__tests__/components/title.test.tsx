import { render } from "@testing-library/react";
import React from "react";
import TitleComponent from "@/app/components/title";

test("Component renders", () => {
  const { asFragment } = render(<TitleComponent title={"Test Title page"} />);

  expect(asFragment()).toMatchSnapshot();
});
