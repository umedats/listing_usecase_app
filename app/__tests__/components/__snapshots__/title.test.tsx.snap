// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-13mqtjd-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <h6
        class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
      >
        Test Title page
      </h6>
    </div>
  </div>
</DocumentFragment>
`;
